### ISSUE TITLE
Switch extra data to tarball

### ISSUE THREAD

**Issue by Barthalion** [flathub/com.visualstudio.code#2]
It would be much more reliable to use tarball for extra data, as it has reliable URL scheme pointing to specific versions.

```
https://vscode-update.azurewebsites.net/1.18.1/linux-x64/stable
```

There is also a tarball for i686, so we could support both arches.

**Comment by nedrichards** [comment]
Yep, switching to tar.gz sounds like a win. I thought the MS link would be stable, shows what I know.

I'm still stuck on an older version of flatpak-builder for now so I can't test a transition to the new extra-data form myself (which would allow the i686 tarball as well). Patches welcome!

**Comment by nedrichards** [comment]
We still use the deb for now, but I switched to the new extra-data format to support i686 as well in https://github.com/flathub/com.visualstudio.code/commit/e5a2ded3c872bc4352a1a1f2894ac161fc1ebe6d

**Comment by nedrichards** [comment]
fixed in #10
